src/sentry/search/events/datasets/semver_and_stage_aliases.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import List, Optional
+
+from snuba_sdk.conditions import Condition, Op
+
+from sentry.api.event_search import SearchFilter
+from sentry.exceptions import InvalidSearchQuery
+from sentry.models import Environment, Release, SemverFilter
+from sentry.search.events.constants import (
+    MAX_SEARCH_RELEASES,
+    OPERATOR_NEGATION_MAP,
+    OPERATOR_TO_DJANGO,
+    SEMVER_EMPTY_RELEASE,
+)
+from sentry.search.events.filter import _flip_field_sort, handle_operator_negation, parse_semver
+from sentry.search.events.types import WhereType
+
+
+class SemverAndStageFilterConverterMixin(ABC):
```

Comment:
Usually we combine names in mixin without `And` e.g. 
```
[RetrieveUpdateDestroyAPIView](https://www.django-rest-framework.org/api-guide/generic-views/#retrieveupdatedestroyapiview)
```


also a good habit to use `abc.ABC`, similar https://github.com/getsentry/sentry/blob/8f00c5b3f1143695c64d1808901b2102d56f18a9/src/sentry/search/events/datasets/base.py#L10-L10

So to summarise:
```diff
class SemverStageFilterConverterMixin(abc.ABC):
```

Reply:
I mean in sentry codebase:
```
(.venv) ➜  sentry git:(master) rg "abc.ABC" | wc -l
      46
```

and without:
```
rg "\(ABC\)"
src/sentry/snuba/metrics/fields/base.py
384:class MetricExpressionBase(ABC):

src/sentry/snuba/outcomes.py
58:class Field(ABC):

src/sentry/release_health/metrics_sessions_v2.py
150:class Field(ABC):

src/sentry/integrations/message_builder.py
13:class AbstractMessageBuilder(ABC):

src/sentry/utils/performance_issues/performance_detection.py
160:class PerformanceDetector(ABC):

tests/sentry/utils/test_services.py
10:class Operation(ABC):
```